src/Command/DoPopulateIndex.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Valantic\ElasticaBridgeBundle\Command;
+
+use Elastica\Index as ElasticaIndex;
+use Pimcore\Model\Element\AbstractElement;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Valantic\ElasticaBridgeBundle\Exception\Command\DocumentFailedException;
+use Valantic\ElasticaBridgeBundle\Index\IndexInterface;
+use Valantic\ElasticaBridgeBundle\Repository\ConfigurationRepository;
+use Valantic\ElasticaBridgeBundle\Repository\DocumentRepository;
+use Valantic\ElasticaBridgeBundle\Repository\IndexRepository;
+use Valantic\ElasticaBridgeBundle\Service\DocumentHelper;
+
+class DoPopulateIndex extends BaseCommand
+{
+    private const OPTION_CONFIG = 'config';
+    private const OPTION_INDEX = 'index';
+    private const OPTION_BATCH_NUMBER = 'batch-number';
+    private const OPTION_LISTING_COUNT = 'listing-count';
+    private const OPTION_DOCUMENT = 'document';
```

Comment:
How do you feel about making these `public` and using them in the `new Process` call in `PopulateIndex`? Same refactoring would also apply to `Index` and `PopulateIndex`.

Reply:
I'm going to move this into the BaseCommand

Reply:
Just moved this to CommandConstants.php